commands/rm.go

```diff
@@ -60,6 +60,9 @@ func runRm(dockerCli command.Cli, in rmOptions) error {
 		return err
 	}
 	if ng != nil {
+		if ng.Name == "default" {
```

Comment:
Is this "default" case or builder with driver "docker"

Reply:
Hum yes so in my mind it is the default builder, whatever driver we use because looking at standalone mode without docker, "default" builder is created after using `ls`:

```dockerfile
FROM alpine
COPY --from=docker/buildx-bin / /usr/bin/
```

```
$ docker buildx build --tag buildx --load .
$ docker run --rm -it buildx sh
/ # apk add tree

/ # tree -a /root
/root
└── .ash_history

0 directories, 1 file

/ # buildx ls
NAME/NODE DRIVER/ENDPOINT  STATUS PLATFORMS
default * docker-container
  default default          Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

/ # tree -a /root
/root
├── .ash_history
└── .docker
    └── buildx
        ├── .lock
        ├── current
        ├── defaults
        └── instances

4 directories, 3 files

/ # buildx rm default

/ # buildx ls
NAME/NODE DRIVER/ENDPOINT  STATUS PLATFORMS
default * docker-container
  default default          Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

I guess this is linked to the store pkg?

Reply:
> builder with driver "docker"

Added another condition to disallow removing context builders which is not working atm anyway and should be done with `docker context rm`.